react/react2/week1/github-users/src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import "./App.css";
+import UserList from "./components/UserList";
+import UserContextProvider from "./providers/UserContextProvider";
+
```

Comment:
.tsx is used with [Typescript](https://stackoverflow.com/questions/64343698/what-is-the-difference-between-js-tsx-and-jsx-in-react) but it looks like you are still dynamically typing this code

Reply:
It seems to infer types from @types/react and @types/react-dom automatically in my case. So I did only custom typing, for some elements...